.bazelci/presubmit.yml

```diff
@@ -42,8 +42,9 @@ tasks:
     # rust-lld isn't available on RBE
     - "-@examples//hello_world_wasm:hello_world_wasm_test"
   windows:
-    windows_targets: &windows_targets
+    build_flags:
     - "--enable_runfiles" # this is not enabled by default on windows and is necessary for the cargo build scripts
+    windows_targets: &windows_targets
```

Comment:
Do we plan to have multiple windows run? Maybe there is no need for the windows_targets indirection?

Reply:
It's to add test_targets, I'm still trying to fix rustdoct_test rules to be able to add them. But I prefer the change to go in first without this.

Reply:
Ok no worries, I am going to merge this PR as-is this is already a huge improvement over the status-quo.